sites/partners/pages/sign-in.tsx

```diff
@@ -32,18 +40,21 @@ const SignIn = () => {
     } catch (err) {
       const { status } = err.response || {}
       if (status === 401) {
-        console.warn(err.message)
-        setError("authentication", {
-          type: "manual",
-          message: t("authentication.signIn.cantFindAccount"),
+        setRequestError({
+          title: t("authentication.signIn.enterValidEmailAndPassword"),
+          content: t("authentication.signIn.afterFailedAttempts"),
+        })
+      } else if (status === 429) {
+        setRequestError({
+          title: t("authentication.signIn.accountHasBeenLocked"),
+          content: t("authentication.signIn.youHaveToWait"),
         })
       } else {
         console.error(err)
-        setError("authentication", {
-          type: "manual",
-          message: `${t("authentication.signIn.error")} ${t(
-            "authentication.signIn.errorGenericMessage"
-          )}`,
+
+        setRequestError({
+          title: t("authentication.signIn.error"),
+          content: t("authentication.signIn.errorGenericMessage"),
```

Comment:
@dominikx96 ,
For this piece and the same logic for public, can you create a login error handler in shared-helpers that they both can use?

Reply:
I would add, as much as we can de-dup any common auth screens between the two apps, that would be ideal. 👍